docs_user/modules/proc_migrating-databases-to-mariadb-instances.adoc

```diff
@@ -137,12 +140,23 @@ apiVersion: v1
 kind: Pod
 metadata:
   name: mariadb-copy-data
+ifeval::["{build_variant}" == "ospdo"]
+  namespace: <OSPDO_NAMESPACE>
+endif::[]
   annotations:
     openshift.io/scc: anyuid
+ifeval::["{build_variant}" != "ospdo"]
```

Comment:
use env vars for annotations, declare that in the tests code and docs, keep things aligned please.
The docs only conditions do not work for tests, we must use env vars instead (so you can apply the docs only conditions for selecting the env var value instead)

Reply:
not sure what you mean here

Reply:
@bogdando can you give an example here ? what change is needed ?
make "ospdo" into a "$NAMESPACE" ?

Reply:
as we agreed, let's remove the namespaces related changes alltogether from this patch scope.
We can discuss the linked examples in 856 then

Reply:
ok , I'll remove the namespace , but this change the annotation (k8s.v1.cni.cncf.io/networks) for opdo envs must be set here,  or it will not work

Reply:
I see. Then please follow the linked example env var to define annotations https://github.com/openstack-k8s-operators/data-plane-adoption/blob/588171f9bd3dcd60cd66af1d656c7ee54a877e75/docs_user/modules/proc_retrieving-topology-specific-service-configuration.adoc?plain=1#L93-L105